### PR TITLE
Man0s/security policy

### PR DESCRIPTION
**Pull request by losman0s**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+2. Attacks requiring access to leaked keys/credentials.
+3. Attacks requiring access to privileged addresses (governance, admin).
+4. Incorrect data supplied by third party oracles (this does not exclude oracle manipulation/flash loan attacks).
+5. Lack of liquidity.
+6. Third party, off-chain bot errors (for instance bugs with an arbitrage bot running on the smart contracts).
+7. Best practice critiques.
+8. Sybil attacks.
+9. Attempted phishing or other social engineering attacks involving marginfi contributors or users
+10. Denial of service, or automated testing of services that generate significant traffic.
+11. Any submission violating Immunefi's rules
```
Comment (jkbpvsc) [inline SECURITY.md:39]: Should we change `Immunefi's` to something else here?

**Review by jkbpvsc — COMMENTED** [review]
Looks good to me, just have a single question in a comment above.

File: SECURITY.md (hunk 1 above)
Comment (losman0s) [reply SECURITY.md:39]: We can scrap this entirely, this list is explicitly non-exhaustive anyway.

Comment (jkbpvsc) [reply SECURITY.md]: Sure lets do it.

**Review by jkbpvsc — APPROVED** [review]

**Pull request merged**